### PR TITLE
Made USD default currency

### LINKED ISSUE #45: Set USD as default value according to task
USD is 17th in rates array

### PR DESCRIPTION
**Pull request by owlerxan**
Closes #45 
- set default currency
- propagated it to fragments

### CODE REVIEW

File: app/src/main/java/com/stonks/OnboardingActivity.kt [hunk 1]
```diff
@@ -36,6 +36,7 @@ class OnboardingActivity : AppCompatActivity() {
             spinnerDefaultCurrency = findViewById(R.id.default_currency_selector_spinner)
             groupLanguageSelection = findViewById(R.id.language_selection_group)
 
+            spinnerDefaultCurrency.setSelection(17)
```
Comment (kramarov-evg) [inline app/src/main/java/com/stonks/OnboardingActivity.kt]: At least make it a class-wide constant

File: app/src/main/java/com/stonks/ui/cryptocurrency/CryptoFragment.kt [hunk 2]
```diff
@@ -21,6 +21,7 @@ import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.schedulers.Schedulers
 import kotlinx.android.synthetic.main.fragment_crypto.*
+import kotlinx.android.synthetic.main.fragment_currency.*
```
Comment (kramarov-evg) [inline app/src/main/java/com/stonks/ui/cryptocurrency/CryptoFragment.kt]: Not needed

File: app/src/main/java/com/stonks/ui/main/MainActivity.kt [hunk 3]
```diff
@@ -19,18 +19,22 @@ import kotlinx.android.synthetic.main.activity_main.*
...
 
-    // Variable storing currently selected fragment for displaying
-    private var selectedFragment: Fragment = CurrencyFragment(bottomNavigationHeight)
-
     private lateinit var rlGuideFirstPage: RelativeLayout
+
     private lateinit var rlGuideSecondPage: RelativeLayout
     private lateinit var bottomNavigation: BottomNavigationView
 
+    private var defaultCurrencyInd = 0
```
Comment (kramarov-evg) [inline app/src/main/java/com/stonks/ui/main/MainActivity.kt]: Why is it 0?

File: app/src/main/java/com/stonks/ui/main/MainActivity.kt [hunk 4]
```diff
@@ -19,18 +19,22 @@ import kotlinx.android.synthetic.main.activity_main.*
...
+
+    // Variable storing currently selected fragment for displaying
+    private lateinit var selectedFragment: Fragment
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        defaultCurrencyInd = intent.getIntExtra("defaultCurrencyInd", 0)
```
Comment (kramarov-evg) [inline app/src/main/java/com/stonks/ui/main/MainActivity.kt]: Also here 0. Why?

**Review by kramarov-evg — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/stonks/OnboardingActivity.kt (hunk 1 above)
Comment (owlerxan) [reply app/src/main/java/com/stonks/OnboardingActivity.kt]: Set to constant

File: app/src/main/java/com/stonks/ui/main/MainActivity.kt (hunk 3 above)
Comment (owlerxan) [reply app/src/main/java/com/stonks/ui/main/MainActivity.kt]: changed to constant

File: app/src/main/java/com/stonks/ui/main/MainActivity.kt (hunk 4 above)
Comment (owlerxan) [reply app/src/main/java/com/stonks/ui/main/MainActivity.kt]: Same here

**Review by kramarov-evg — APPROVED** [review]
Nicely done

**Pull request merged**